infra/dlv/README.md

```diff
@@ -0,0 +1,90 @@
+# DLV Debugging of VCH servers
+
+## Overview
+
+This notes describes how to set up dlv remote debugging of VCH servers
+
+## Building debug enabled binaries (non stripped)
+
+Set the following environment variable
+``` shell
+export VIC_DEBUG_BUILD=true
+```
+this tells the makefile to build non stripped binaries
```

Comment:
nit: I think documentation is more readable if code-snippets don't split sentences. You could avoid that in this case by re-writing as:
```markdown

Set the following environment variable to tell the Makefile to build non-stripped binaries:
``` shell
export VIC_DEBUG_BUILD=true
```